src/test/scala/com/amazon/deequ/VerificationSuiteTest.scala

```diff
@@ -57,24 +56,28 @@ class VerificationSuiteTest extends WordSpec with Matchers with SparkContextSpec
           CheckStatus.Success)
         assert(VerificationSuite().onData(df).addCheck(checkToErrorOut).run().status ==
```

Comment:
refactor VerificationSuite().onData(df).addCheck(checkToErrorOut).run().status into a temp variable  (also for the cases below)